DirectEmployers/MyJobs
# [PD-3579] dropdown combos when switching pages

## What's this PR do?
PR changes the state to not show the multifilter dropdown if it is being shown when switching between dimensions

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside the site traffic analytics page with the multifilter dropdowns

## How should this be manually tested?
Pull down the repository. Go into the analytics application and go to a multifilter page. When you open it, switch dimensions to another multifilter page and it should be closed now do to the state change

## Any background context you want to provide?
